tests/serving/test_remote.py

```diff
@@ -234,19 +234,31 @@ def test_remote_advance(httpserver, engine):
             url=httpserver.url_for("/"),
             url_expression="endpoint + event['url']",
             body_expression="event['data']",
+            headers_expression="event['body_headers']",
             input_path="req",
             result_path="resp",
         )
-    ).to(name="s3", handler="echo").respond()
+    ).to(name="s3", handler="echo", full_event=True).respond()
 
     server = function.to_mock_server()
     try:
-        resp = server.test(body={"req": {"url": "/dog", "data": {"x": 5}}})
+        resp = server.test(
+            body={
+                "req": {
+                    "url": "/dog",
+                    "data": {"x": 5},
+                    "body_headers": {"Test-header": "true"},
+                }
+            }
+        )
     except Exception as e:
         raise e
     finally:
         server.wait_for_completion()
+    # resp_body = resp['body']
+    resp__headers = resp["req"].pop("body_headers")
```

Comment:
```suggestion
    resp_headers = resp["req"].pop("body_headers")
```

Reply:
i wanted to remove the `body_headers` from the response so if i do not using `pop` the headers will be keep in the response

Reply:
Why do you want to remove it? It seems inconsequential.

Reply:
the reason i wanted to remove the headers is because if i do not do it this test fail - resp == {"req": {"url": "/dog", "data": {"x": 5}}, "resp": {"post": "ok"}} , becuase the headers are in the {"req": and their values are not constant.
i thought to remove the headers and then keep the old test and check the only add key for the headers